android/src/main/kotlin/com/github/picture2pc/android/viewmodel/camerascreenviewmodels/CameraViewModel.kt

```diff
@@ -50,14 +51,20 @@ class CameraViewModel(
         pictureManager.takeImage()
     }
 
-    fun sendImage() {
-        viewModelScope.launch {
-            dataTransmitter.sendPicture(
-                TcpPayload.Picture(
-                    getLastImage().toByteArray(),
-                    lastCorners
+    fun sendImage(context: android.content.Context) {
+        val launch = viewModelScope.launch {
+                dataTransmitter.sendPicture(
```

Comment:
The `sendPicture` funtkion returns a bool, true if successfull and false if not

Reply:
you would then need to return it like `return@async dataTransmittter...`

Reply:
![Bild_2024-10-16_001531488](https://github.com/user-attachments/assets/3f3eee91-7bbf-421f-91d2-15c9c8363a9f)
sendPicture returns Unit not bool, no? Could i add a boolean parameter in the sendPicture or something?

Reply:
Nvm i think just checking for void/Unit might work

Reply:
if this fucking netcode would work properly id be very happy <3